index.html

```diff
@@ -263,7 +263,24 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
     </div>
   </div>
 
-
+  <div id="form">
+    <p id="form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <form id="formsubmit" action="https://formspree.io/f/xrgdnkkd"
+     method="POST">
+      <label>
+      <input class="form_inputs" type="text" placeholder="Full name" maxlength="30" required>
+      </label>
+      <div class="divide"><img src="./assets/Vector 4.png" alt=""></div>
+      <label>
+      <input class="form_inputs" type="email" placeholder="Email address" name="email" required>
+      </label>
+      <div class="divide"><img src="./assets/Vector 4.png" alt=""></div>
+      <label>
+      <textarea id="form_text_area" placeholder="Enter text here" name="message" maxlength="500" required></textarea>
+      </label>
+      <button id="form_button" class="orange_btns" type="submit"><p id="action_btn_text"> Get in touch</p></button>
```

Comment:
- [ ] The form is working fine and the style is very similar to the Figma template. But there is no need for the `label`   around the `input` and `textarea` tags in this form. The `label` is used to add a label on top of an `input` but the template doesn't use labels, only the placeholder text to inform the user what is the input about. Kindly remove the `label` tags.
 - [ ] Just like the `label` there is no need to add a `p` tag inside a `button` tag. You can write text normally inside the `button` tag and it is not a good practice to add tags inside the `button` tag. Kindly remove the `p` tag.

Reply:
Removed label tags, removed <p> tags and added CSS to compensate